doc/README.adoc

```diff
@@ -1,5 +1,17 @@
 = Kadalu Documentation
 
+
+== Talks and Blog posts
+
+1. [Blog] link:https://medium.com/@tumballi/glusters-management-in-k8s-13020a561962[Gluster��s management in k8s]
+2. [Blog] link:https://aravindavk.in/blog/gluster-and-k8s-portmap/[Gluster and Kubernetes - Portmap]
+3. [Talk] link:rethinking-gluster-management-using-k8s.pdf[Rethinking Gluster Management using k8s]
+4. [Demo] link:https://asciinema.org/a/259949[Asciinema recording - Kadalu Setup]
+5. [Demo] link:https://asciinema.org/a/259951[Asciinema recording - KaDalu CSI to claim Persistent Volumes]
+
```

Comment:
- Will it be good to mention this as well https://thoughtexpo.com/tags/kadalu/?
- No problem whatsoever if it's not mentioned

Reply:
Sure... makes sense to add.